docs/src/app/components/pages/discover-more/Showcase.js

```diff
@@ -22,6 +22,12 @@ const appList = [
   //   link: '',
   // },
   {
+    title: 'Three.js Live - Interact with WebGL code in your browser',
```

Comment:
We add the demo chronologically for now. Could add yours at the end of the list?
Thanks. We might reorder them for the `next` branch.